### ISSUE TITLE
enterpriseAuthentication and domain administrator 

### ISSUE THREAD

**Issue by DanieleScipioni** [MicrosoftDocs/windows-uwp#1679]
In the description of enterpriseAuthentication capability is written

&gt; This capability will require a domain administrator to enable the functionality for all apps.

What exactly does it means? Which option the domain administrator must change and how? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 606c0ecf-783c-5f07-8019-cdd505ee1359
* Version Independent ID: 05430a91-280f-bd65-23d0-4d5f912f5b3c
* Content: [Networking basics - Windows UWP applications](https://docs.microsoft.com/en-us/windows/uwp/networking/networking-basics#feedback)
* Content Source: [windows-apps-src/networking/networking-basics.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/networking/networking-basics.md)
* Product: **uwp**
* Technology: **communication**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Thanks, @DanieleScipioni, I've published an update to the topic now. I asked a few engineers and although we had some theories for what that sentence might mean, we weren't certain. Also, there's another topic that explains this same capability in more detail. So, in the interest of clarity, I've removed the confusing sentence and I've added a link to the other topic. If you look at the Networking basics topic again, in the table next to enterpriseAuthentication, you'll find the link to the other topic.

Hope that helps!
~Steve
